content/get-started/workshop/04_sharing_app.md

```diff
@@ -38,7 +38,8 @@ In the following image, you can see an example Docker command from Docker Hub. T
 
 ## Push the image
 
-1. In the command line, run the `docker push` command that you see on Docker
+1. Sign in to Docker Hub using the command: `docker login YOUR-USER-NAME YOUR-PASSWORD`.
```

Comment:
```suggestion
1. Sign in to Docker Hub using the command: `docker login`.
```

Slightly tricky; depending on the situation, this will ask for a username (and password), or (with current versions of the CLI) will start a web-based OAuth flow;

```bash
docker login

USING WEB-BASED LOGIN

i Info → To sign in with credentials on the command line, use 'docker login -u <username>'


Your one-time device confirmation code is: VQXV-KGKD
Press ENTER to open your browser or submit your device code here: https://login.docker.com/activate

Waiting for authentication in the browser…
```

So, ... perhaps just `docker login` would be enough?